dataikuapi/dss/webapp.py

```diff
@@ -0,0 +1,90 @@
+import sys
+from .future import DSSFuture
+
+if sys.version_info >= (3,0):
+  import urllib.parse
+  dku_quote_fn = urllib.parse.quote
+else:
+  import urllib
+  dku_quote_fn = urllib.quote
+
+
+class DSSWebApp(object):
+    """
+    A handle to manage a webapp
+    """
+    def __init__(self, client, project_key, webapp_id, state=None):
+        """Do not call directly, use :meth:`dataikuapi.dss.project.DSSProject.get_webapps`"""
+        self.client = client
+        self.project_key = project_key
+        self.webapp_id = webapp_id
+        self.state = state
+
+    def get_state(self):
```

Comment:
make a call to fetch state if self.state is None. We can't demand that you only use DSSWebapp objects coming from the list() call

and the state is important, people will use it to monitor what is running or not...